scripts/translate_locales.py

```diff
@@ -0,0 +1,33 @@
+
+# A script to generate a template which will be used to localize the supported locale name
+# For more inofrmation see https://bugzil.la/859499#c11
+
+from django.conf import settings
+
+from jinja2 import Environment
+
+
+template_string = ("""
+
+This template is automatically generated by scripts/translate_locales.py to make the languages name localizable. Do not edit it manually
+Background: https://bugzil.la/859499#c11
+
+{% for lang in languages %}
+   {{ '{{ _("' + lang + '") }}' }}
+{% endfor %}
```

Comment:
This template has too much whitespace between the locale names. I think changing to ``{%- endfor %}`` (adding a minus sign at the beginning) will get rid of the extra new line.